activerecord/test/cases/adapters/postgresql/connection_test.rb

```diff
@@ -151,7 +151,7 @@ def test_statement_key_is_logged
     # When prompted, restart the PostgreSQL server with the
     # "-m fast" option or kill the individual connection assuming
     # you know the incantation to do that.
-    # To restart PostgreSQL 9.1 on OS X, installed via MacPorts, ...
+    # To restart PostgreSQL 9.1 on macOS, installed via MacPorts, ...
```

Comment:
Was thinking about it, but there are a bunch of other places that mention MacPorts, and I figured it would be better to figure that out separately